packages/app/src/styles/styleFragments.ts

```diff
@@ -31,6 +35,18 @@ export const V = {
       borderWidth: { _: 2 },
       transition: `all 0.3s`,
     },
+    floating:{
+      position:`absolute`,
+      bottom:`5vh`,
+      right:`10vh`,
+      bg: { _: `background`, hover: `accent` },
+      color: { _: `accent`, hover: `background` },
+      borderRadius: `full`,
+      borderWidth: { _: 2 },
+      transition: `all 0.3s`,
+      outline:{_:`none`,focus:`none`},
+      fontSize:`25px`
```

Comment:
```suggestion
      fontSize: `xl`,
```
I'm not sure `xl` would be as big as `25px`, but make sure you use pre-defined rem unit fontSizes. They are here: https://github.com/gregberge/xstyled/blob/main/packages/system/src/defaultTheme.ts#L318

Reply:
I changed `25px` to `xl`. 
I will make sure use pre-defined rem unit.
thanks.